src/ai/backend/client/cli/admin/scaling_group.py

```diff
@@ -98,7 +98,10 @@ def list(ctx: CLIContext) -> None:
     "--scheduler-opts",
     type=JSONParamType(),
     default="{}",
-    help="Set scheduler options as a JSON string.",
+    help="""
+        Set scheduler options as a JSON string.
+        For 'allowed_session_types', if the key does not exist, the values 'interactive' and 'batch' are set.
```

Comment:
Let's prefer the active voice.
```suggestion
        If the 'allowed_session_types' key is not specified, the policy defaults to accept both 'interactive' and 'batch'.
```